packages/components/organisms/f-checkout/src/components/tests/Header.test.js

```diff
@@ -0,0 +1,148 @@
+import Vuex from 'vuex';
+import { VueI18n } from '@justeat/f-globalisation';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Header from '../Header.vue';
+import { i18n, defaultState, createStore } from './helpers/setup';
+import EventNames from '../../event-names';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+describe('Header', () => {
+    allure.feature('Checkout-Header');
+
+    const propsData = {
+        loginUrl: 'http://localhost/login'
+    };
+
+    it('should be defined', () => {
+        // Arrange
+        const wrapper = shallowMount(Header, {
+            store: createStore(),
+            i18n,
+            localVue,
+            propsData
+        });
+
+        // Assert
+        expect(wrapper).toBeDefined();
+    });
+
+    describe('computed ::', () => {
+        describe('isLoggedIn ::', () => {
+            describe('when set to true', () => {
+                const wrapper = shallowMount(Header, {
+                    store: createStore({ ...defaultState, isLoggedIn: true }),
+                    i18n,
+                    localVue,
+                    propsData
+                });
+
+                it('should render the user header', () => {
+                    // Arrange
+                    const userHeader = wrapper.find('[data-test-id="user-header"]');
+
+                    // Assert
+                    expect(userHeader).toBeDefined();
+                });
+
+                it('should show the login link', () => {
```

Comment:
This name could be improved to make it more accurate. If you compare it with the test for guest users, this one says "login link" and the other one says "login button". But I think there are bigger differences between them. One is for switching users, the other one is logging in or registering.